cms/djangoapps/models/settings/course_metadata.py

```diff
@@ -46,6 +46,10 @@ class CourseMetadata(object):
         'language',
         'certificates',
         'minimum_grade_credit',
+        'default_time_limit_mins',
```

Comment:
Does ``mins`` here mean ``minimums`` or ``minutes``? I'm assuming the latter, but I'd prefer if it were spelled out.